src/components/Request/FormFields/AdvertisementLength.tsx

```diff
@@ -0,0 +1,65 @@
+import { Label, Radio, RadioGroup, Text } from "@fluentui/react-components";
+import {
+  Control,
+  Controller,
+  FieldErrors,
+  UseFormSetValue,
+} from "react-hook-form";
+import "../Request.css";
+import { RHFRequest } from "../NewRequestForm";
+import { RadioButtonFilled } from "@fluentui/react-icons";
+
+interface IAdvertisementLength {
+  name: string;
+  control: Control<RHFRequest, unknown>;
+  errors: FieldErrors<RHFRequest>;
+  setValue: UseFormSetValue<RHFRequest>;
+}
+
+const PaySystem = (adLength: IAdvertisementLength) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
```

Comment:
While the styling varies slightly from SharePoint, should we consider using the new "<Field>" component from FluentUI, which provides a validation message, hint, label, etc?  https://react.fluentui.dev/?path=/docs/components-field--default